buildpack.toml

```diff
@@ -55,6 +55,16 @@ api = "0.7"
     optional = true
     version = "2.7.0"
 
+  [[order.group]]
+    id = "paketo-buildpacks/yarn"
+    optional = true
+    version = "1.0.0"
+
+  [[order.group]]
+    id = "paketo-buildpacks/node-engine"
+    optional = true
+    version = "1.1.0"
+
```

Comment:
I think these should com after the JVM provider buildpack. Technically closure tools and leinigen are build related as well and could eventually support integration with Node also.

Reply:
That's a good point! 👍